src/telemetry.rs

```diff
@@ -7,6 +7,25 @@ use opentelemetry_sdk::{
 
 use crate::SubscriberConfig;
 
+fn normalize_collector_url(collector_url: &str) -> String {
+    // OTLP before version 0.15 didn't append a /v1/traces suffix, but started doing so there.
+    // For backwards compatibility we strip it from configurations that do have it.
+    let collector_url = collector_url
+        // In case of a trailing slash strip it
+        .strip_suffix('/')
+        .unwrap_or(collector_url)
+        .strip_suffix("/v1/traces")
+        .unwrap_or(collector_url);
+
+    // Backport https://github.com/open-telemetry/opentelemetry-rust/pull/1553
+    let collector_url = collector_url.strip_suffix('/').unwrap_or(collector_url);
```

Comment:
I don't understand why is needed to strip the suffix twice..

Reply:
I dunno, it was already there, I'll remove it

Reply:
Fixed

Reply:
ah no sorry it does need to be stripped, my bad